kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/domain-custom-resource-template.yaml

```diff
@@ -35,6 +35,13 @@ spec:
   asName: "%ADMIN_SERVER_NAME%"
   # The Admin Server's ListenPort
   asPort: %ADMIN_PORT%
+  # Whether to include the server out file into the pod's stdout
+  includeServerOutInPodLog: %INCLUDE_SERVER_OUT_IN_POD_LOG%
+  # The in-pod name of the directory to store the domain, node manager, server logs, and server .out
+  # files in.
+  # If not specified or empty, then these files will be stored in their normal locations under the
+  # domain home.
```

Comment:
If not specified, these files will be stored in /shared/logs/[domain-uid].   If empty, in their default location.   Otherwise in the specified location.